### PR TITLE
fix: Update Insights tab visual clue name

### PR DESCRIPTION
**Pull request by MrSltun**
### Description
Follow up on #6851
Update the visual clue name for the Insights tab 

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- update insights tab visual clue name - mrsltun

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Review by olerichter00 — CHANGES_REQUESTED** [review]

File: src/app/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export const MyProfileHeaderMyCollectionAndSavedWorks: React.FC<{
               jsx: (
                 <VisualClueDot style={{ marginLeft: 5, alignSelf: "flex-start", marginTop: 1 }} />
               ),
-              visualClueName: "AddArtworkWithInsights",
+              visualClueName: "AddedArtworkWithInsightsMessage_MyCTab",
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/app/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tsx:50]: Or is this when a user adds an artwork from the My Collection tab and just in this specific case wants to see the blue dot?

**Review by olerichter00 — DISMISSED** [review]

File: src/app/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MrSltun) [reply src/app/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tsx:50]: no wait, we should add two clues, moving to the Insights tab doesn't close the message

[the remark this answers is not in the document]
Comment (olerichter00) [reply src/app/Scenes/MyProfile/MyProfileHeaderMyCollectionAndSavedWorks.tsx:50]: Yes, let's probably add an additional clue 🔬

File: src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx [hunk 2]
```diff
@@ -387,6 +387,9 @@ const addArtworkMessages = async ({
         ? "AddedArtworkWithInsightsMessage_MyCTab"
         : "AddedArtworkWithInsightsMessage_InsightsTab"
     )
+    if (sourceTab === Tab.collection) {
+      addClue("AddedArtworkWithInsightsVisualClueDot")
+    }
```
Comment (MrSltun) [inline src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx:392]: or like this 👀 @olerichter00 
```js
    if (sourceTab === Tab.collection) {
      addClue("AddedArtworkWithInsightsMessage_MyCTab")
      addClue("AddedArtworkWithInsightsVisualClueDot")
    } else {
      addClue("AddedArtworkWithInsightsMessage_InsightsTab")
    }
```

Comment (olerichter00) [inline src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx:392]:
```suggestion
    if (sourceTab === Tab.collection) {
      addClue("AddedArtworkWithInsightsMessage_MyCTab")
      addClue("AddedArtworkWithInsightsVisualClueDot")
    } else {
      addClue("AddedArtworkWithInsightsMessage_InsightsTab")
    }
```

Comment (olerichter00) [reply src/app/Scenes/MyCollection/Screens/ArtworkForm/MyCollectionArtworkForm.tsx:392]: perfect 👍

**Review by olerichter00 — DISMISSED** [review]

**Pull request merged**